librocksdb_sys/crocksdb/rocksdb/c.h

```diff
@@ -111,6 +111,32 @@ typedef struct crocksdb_ingestexternalfileoptions_t crocksdb_ingestexternalfileo
 typedef struct crocksdb_sstfilewriter_t   crocksdb_sstfilewriter_t;
 typedef struct crocksdb_ratelimiter_t     crocksdb_ratelimiter_t;
 typedef struct crocksdb_pinnableslice_t   crocksdb_pinnableslice_t;
+typedef struct crocksdb_user_collected_properties_t
+    crocksdb_user_collected_properties_t;
+typedef struct crocksdb_table_properties_collection_t
+    crocksdb_table_properties_collection_t;
+typedef struct crocksdb_table_properties_collector_factory_t
+    crocksdb_table_properties_collector_factory_t;
+
+typedef struct crocksdb_table_properties_collector_context_t {
+    void* collector;
+    const char* (*name)(void*);
```

Comment:
It feels strange `void*` here refers to `crocksdb_table_properties_collector_context_t`. Conventionally, `void*` should refers to `collector`.